dss-api.yml

```diff
@@ -673,9 +673,12 @@ paths:
     put:
       security:
         - dcpAuth: []
-      summary: Create a event subscription.
+      summary: Create an event subscription.
       description: >
         Register an HTTP endpoint that is to be notified when a given event occurs.
+
+        Each user is allowed 100 subscriptions, a limit that may be increased in the future. Concerns about
+        notification service limitations should be routed to the DSS development team.
```

Comment:
@kislyuk Do you have any comments on our documentation of subscription limitations?